website/docs/getting-started/setup/instance-management/README.mdx

```diff
@@ -1,115 +1,15 @@
 ---
-
+description: Efficiently manage your Appsmith instances.
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Instance Management
 
-## Checkpoint version and upgrades
-
-When you upgrade manually, you must upgrade to the checkpoint version released between your version and the latest version.
-
-Appsmith checkpoint versions:
-* v1.9.2
-
-
-Access your Appsmith instance and make the following change to the docker-compose.yml file:
-
-<Tabs groupId="operating-systems">
-  <TabItem value="CE" label="Community Edition" default>
-  
-    From:
-
-```bash
-services:
-  appsmith:
-    image: index.docker.io/appsmith/appsmith-ce
-    container_name: appsmith
-```
-
-To:
-
-```bash
-services:
-  appsmith:
-    image: index.docker.io/appsmith/appsmith-ce:v1.9.2
-    container_name: appsmith
-```
-
-
-</TabItem>
-   <TabItem value="BE" label="Business Edition">
-   
-From:
-
-```bash
-services:
-  appsmith:
-    image: index.docker.io/appsmith/appsmith-ee
-    container_name: appsmith
-```
-
-To:
-
-```bash
-services:
-  appsmith:
-    image: index.docker.io/appsmith/appsmith-ee:v1.9.2
-    container_name: appsmith
-```
-   
-   
-   </TabItem>
-</Tabs>
-
-Save this file, then run the following command on your shell to restart Appsmith.
-
-```
-docker-compose up -d
-```
-
-
-With this, your server starts with the older version that makes all the necessary schema changes to your Appsmith instance. Once this process has been completed successfully (you can verify this by logging in to your Appsmith instance), you can return to the original version of your `docker-compose.yml` file and resume normal operations.
-
-
-<Tabs groupId="operating-systems">
-  <TabItem value="CE" label="Community Edition" default>
-
-  ```bash
-  services:
-  appsmith:
-    image: index.docker.io/appsmith/appsmith-ce
-    container_name: appsmith
-  ```
-  
-  </TabItem>
-  <TabItem value="BE" label="Business Edition">
-
-  ```
-services:
-  appsmith:
-    image: index.docker.io/appsmith/appsmith-ee
-    container_name: appsmith
-  ```
-  </TabItem>
-</Tabs>
-
-
-Run the restart command once again: 
-
- ```
- docker-compose up -d
-  ```
-:::note 
-If you haven't been using your Appsmith instance for a while, it could be possible that there might be multiple versions that show up in the error message. In such a case, perform the same steps for each version in order, starting from the oldest.
-:::
-
-
 ## Switching off auto updates
```

Comment:
Isn't this also related to updates? Until we figure out how Instance Management should look, can we not make any changes there? Else we will have make this a proper landing page. I think this should be handled in a different PR. as in the deploy preview this page looks and incomplete.

Reply:
That's a configuration task to turn on or off auto updates so haven't moved it.

Reply:
We can work on the landing page in the incremental PR. But as the checkpoint upgrades were related, moving that page to the appropriate location made more sense. I can add some cards for now to the page so that it doesn't look odd.